### PR TITLE
TECH-938 force exception messages to encode to utf-8

### PR DESCRIPTION
**Pull request by Inanepenguin**
@ColinDKelley here's a fix for the utf8 issue

### CODE REVIEW

File: lib/exception_handling.rb [hunk 1]
```diff
@@ -336,7 +336,7 @@ def alert_warning(exception_or_string, alert_name, exception_context)
       ex = make_exception(exception_or_string)
       log_error(ex, exception_context)
       begin
-        ExceptionHandling::Sensu.generate_event(alert_name, exception_context.to_s + "\n" + ex.message.to_s)
+        ExceptionHandling::Sensu.generate_event(alert_name, exception_context.to_s + "\n" + encode_utf8(ex.message.to_s))
```
Comment (ColinDKelley) [inline lib/exception_handling.rb]: It looks like the `.to_s` isn't really needed before calling `encode_utf8`.

**Comment by ColinDKelley** [comment]
:+1:

**Comment by ColinDKelley** [comment]
@codyjca @primiti  Will this cause a problem if we push this right away?  I know you had some other changes in this area and we might get version number clash.

**Comment by primiti** [comment]
Assuming it can wait for next Wednesday, I think we can push this through.   I want to see how we can help improve this process, so I want to feel the pain myself.  I am happy to bump the version and then chase the dependencies.  Cody gave me a heads up on what would be required...

**Pull request merged**